.drone.yml

```diff
@@ -8,53 +8,30 @@ platform:
   arch: amd64
 
 steps:
-- name: my-build
-  pull: if-not-exists
-  image: docker:18.03
-  commands:
-  - docker build -t hocs_serenity_automation .
-  - docker run hocs_serenity_automation
-  environment:
-    DOCKER_HOST: tcp://172.17.0.1:2375
-  when:
-    branch:
-    - master
-    event:
-    - push
+- name: build project
+    image: quay.io/ukhomeofficedigital/openjdk11:v11.0.5_10
+    commands:
+      - ./gradlew build
+    when:
+      event:
+        - push
 
-- name: image_to_quay
-  pull: if-not-exists
-  image: quay.io/ukhomeofficedigital/drone-docker
+- name: build & push
+  image: plugins/docker
   settings:
     registry: quay.io
-    repo: quay.io/ukhomeofficedigital/hocs-serenity-automation
+    repo: quay.io/ukhomeofficedigital/hocs_serenity_automation
     tags:
-    - build_${DRONE_BUILD_NUMBER}
-    - latest
+      - build_${DRONE_BUILD_NUMBER}
   environment:
     DOCKER_PASSWORD:
-      from_secret: docker_password
+      from_secret: QUAY_ROBOT_PASSWORD
     DOCKER_USERNAME: ukhomeofficedigital+hocs_quay_robot
   when:
-    branch:
-    - master
     event:
-    - push
-
-- name: tag-docker-image-with-git-tag
-  pull: if-not-exists
-  image: docker:17.09.1
-  commands:
-  - docker login -u="ukhomeofficedigital+hocs_quay_robot" -p=$${DOCKER_PASSWORD} quay.io
-  - docker tag hocs-serenity-automation quay.io/ukhomeofficedigital/hocs-serenity-automation:$${DRONE_TAG}
-  - docker push quay.io/ukhomeofficedigital/hocs-serenity-automation:$${DRONE_TAG}
-  environment:
-    DOCKER_HOST: tcp://172.17.0.1:2375
-    DOCKER_PASSWORD:
-      from_secret: docker_password
-  when:
-    event:
-    - tag
+      - push
+  depends_on:
```

Comment:
I'd leave out the `depends_on` stuff for now. If it's defined, Drone will do a cool acyclic dependency graph thing, but if it's not it'll just run through the steps sequentially top to bottom. While you're still trying to figure out why everything's broken it'll just add complexity.

Reply:
Removed.